assets/control-plane/prometheus-rule.yaml

```diff
@@ -525,35 +525,6 @@ spec:
       for: 15m
       labels:
         severity: critical
-  - name: kube-apiserver-histogram.rules
```

Comment:
:+1: 
these rules have been migrated to [cluster-kube-apiserver-operator](https://github.com/openshift/cluster-kube-apiserver-operator/blob/005a95607cf9f8db490e962b549811d8bc0c5eaf/bindata/assets/alerts/kube-apiserver-slos.yaml#L424-L450) so they don't need to live here anymore